src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -463,6 +463,20 @@ - (void)invokeCommandWithEndpointID:(NSNumber *)endpointID
     [self setExpectedValues:expectedValues expectedValueInterval:expectedValueInterval];
 }
 
+- (void)openCommissioningWindowWithSetupPasscode:(NSNumber *)setupPasscode
+                                   discriminator:(NSNumber *)discriminator
+                                        duration:(NSNumber *)duration
+                                     clientQueue:(dispatch_queue_t)clientQueue
+                                      completion:(MTRDeviceOpenCommissioningWindowHandler)completion
+{
+    auto * baseDevice = [[MTRBaseDevice alloc] initWithNodeID:@(self.nodeID) controller:self.deviceController];
```

Comment:
This API is going to change soon, fwiw.  But yes, we can think about a helper.  That said, I'm not sure it would be that much simpler than this line...

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/22843